docs/source/deployment/google_cloud_run.rst

```diff
@@ -103,13 +103,33 @@ This guide uses the IrisClassifier BentoService from the :doc:`Quick start guide
     }
 
 
+After saving the BentoService instance, you can now start a REST API server with the
+model trained and test the API server locally:
```

Comment:
The BentoML saved bundle created can now be used to start a REST API Server hosting the BentoService and available for sending test request: